### PR TITLE
Introduce actix

### PR DESCRIPTION
**Pull request by elegaanz**
With a routing example, but I'm not sure it is the best way to do it (as @fdb-hiroshima suggested on Matrix, we could use proc-macros that are similar to what Rocket does).

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -52,13 +55,15 @@ init_i18n!(
...
+mod api;
 mod mail;
 #[cfg(feature = "test")]
 mod test_routes;
+mod web;
 
 compile_i18n!();
 
-fn main() {
+fn main() -> std::io::Result<()> {
```
Comment (igalic) [inline src/main.rs:66]: What's this do?

File: src/main.rs [hunk 2]
```diff
@@ -79,6 +84,27 @@ and https://docs.joinplu.me/installation/init for more info.
...
+                    // TODO: real error page
+                    aweb::route().to(|| HttpResponse::NotFound()),
+                ),
+        )
+    })
+    .bind("127.0.0.1:7878")?
+    .run()?;
+
     println!("Welcome to the async experiment of Plume.");
     println!("If you can only see this message, we're not done yet.");
```
Comment (igalic) [inline src/main.rs:107]: we could now turn this into a log message now 😅

**Review by igalic — COMMENTED** [review]
❓

File: src/main.rs (hunk 1 above)
Comment (elegaanz) [reply src/main.rs:66]: It allows to use `?` in `main` basically. And if we return an error the program will exit with something else than `0`.

File: src/main.rs (hunk 2 above)
Comment (elegaanz) [reply src/main.rs:107]: I don't added `log` yet! But yes, we should do it at some point.

**Pull request merged**